src/site/_includes/pattern.njk

```diff
@@ -1,5 +1,5 @@
 ---
-layout: patterns
+layout: pattern-page
 pageScripts:
```

Comment:
Any page that shows the tabs widget will need to inline it into the head, otherwise there's a big flash of unstyled content.

![Kapture 2021-08-13 at 16 10 39](https://user-images.githubusercontent.com/1066253/129426183-e82a15b7-8114-47dd-97c1-29a71d1f4a5c.gif)

You can inline script into the head using the `inlineScript` yaml.
https://github.com/GoogleChrome/web.dev/blob/67c579c98bc1e21d1e2989ad459eed80aadc7e6a/src/site/_includes/partials/head.njk#L45-L53

The path needs to be relative to the _includes folder. Here's an example:
https://github.com/GoogleChrome/web.dev/blob/67c579c98bc1e21d1e2989ad459eed80aadc7e6a/src/site/_includes/course.njk#L5-L6

Reply:
I appreciate that inlining works well (and we do CSP it and so on) but I _also_ do feel like `web-header` could have a sensible default set of styles here, since we know how big it will end up being.

Reply:
I'm not sure I understand how it relates to `web-header`? You could try to have default styles but you'll have to make sure to test with a production build to make sure purgeCSS doesn't remove them. I think inlining the component on this page may be simpler.

Reply:
I realized this won't work in this case because the Tabs element has a number of dependencies and we'd have to inline all of those as well, which seems like a lot...

So it seems like having default styles is probably our best bet. My hunch is they should do something like:

```
web-tabs:not(:defined) {
  // make all of the `pre` content inside of tabs invisible
}
```

Reply:
TIL :defined !!